guides/tutorials/codelabs/gce-source-to-prod/index.md

```diff
@@ -20,7 +20,7 @@ You will be using a custom-built image that includes sample application source c
 
 This codelab is an extension of the [Hello World Deployment](../hello-deployment) tutorial. It runs specifically on the Google Cloud Platform, and requires a billing-enabled GCP project.
 
-## Part 0: Environment Setup
+## Part 0: Environment setup
```

Comment:
Unrelated to the capitalization issue, but could you change this to "Set up environment" ? Or even "Set up your environment"